osu.Game.Rulesets.Catch/Objects/Drawables/DrawableFruit.cs

```diff
@@ -41,6 +38,13 @@ private void load()
             HyperDash.BindValueChanged(_ => updatePiece(), true);
         }
 
+        protected override void UpdateInitialTransforms()
+        {
+            base.UpdateInitialTransforms();
+
+            ScaleContainer.Rotation = (RandomSingle(1) - 0.5f) * 40;
```

Comment:
Not sure about having a plain property set in `UpdateInitialTransforms()`. Would probably make this a

```suggestion
            ScaleContainer.RotateTo((RandomSingle(1) - 0.5f) * 40);
```

for consistency at least I guess? There's no other usage that just does plain sets on drawable properties as far as I can tell (there are some that set lifetime, but that's not a transformable property anyway), but I'm not sure if that has any functional difference (it shouldn't from what I know of how transforms work).